exercises/luhn/example.rs

```diff
@@ -1,5 +1,6 @@
 pub fn is_valid(candidate: &str) -> bool {
-    if candidate.chars().any(|c| c.is_alphabetic()) || candidate.chars().count() == 1 {
+    if input.chars().filter(|c| c.is_digit(10)).take(2).count() <= 1 ||
```

Comment:
My bad, copying from my solution (which used input) to example, I forgot to change the name back. Hopefully fixed.